src/backend/gporca/libgpopt/src/xforms/CXformCTEAnchor2TrivialSelect.cpp

```diff
@@ -100,6 +100,14 @@ CXformCTEAnchor2TrivialSelect::Transform(CXformContext *pxfctxt,
 	CExpression *pexprChild = (*pexpr)[0];
 	pexprChild->AddRef();
 
+	if (COperator::EopLogicalCTEAnchor == pexpr->Pop()->Eopid() &&
```

Comment:
the current expression can be not EopLogicalCTEAnchor?

Reply:
removed